lib/routes/xbmu/announcement.ts

```diff
@@ -0,0 +1,101 @@
+// Announcement for Northwestern Minzu University
```

Comment:
Combine the duplicated code instead of having two copies of similar code.

Reply:
Thanks for reviewing.
Because there are maybe some changes between two webpages so I think the cost of encapsulating them into combination is greater than not. So I want to keep some flexibility.